smarts/waymo/waymo_browser.py

```diff
@@ -591,7 +599,7 @@ def save_plot(
         def update(i):
             drawn_pts = []
             for (xs, ys), point in zip(data, points):
-                if i < len(xs):
+                if i < len(xs) and xs[i] and ys[i]:
```

Comment:
I think we need to allow `xs[i]` and `ys[i]` to be 0.0.
```suggestion
                if i < len(xs) and xs[i] is not None and ys[i] is not None:
```

Reply:
Good catch! (facepalm)